docs/examples.md

```diff
@@ -9,7 +9,6 @@ import hid
 
 for device_dict in hid.enumerate():
     keys = list(device_dict.keys())
-    keys.sort()
```

Comment:
This is just an example, but I imagine the `sort()` was call was added because Mac OS returns devices in no particular order.  And, presumably, other platforms could behave the same way.

_(The same comment applies to `try.py`)._

Reply:
I felt the sort was a bit an unnecessary addition. I'll add it it again if you think it's useful.